client/src/app/site/pages/meetings/pages/participants/pages/participant-import/services/participant-import.service/participant-import.service.ts

```diff
@@ -44,7 +44,7 @@ export class ParticipantImportService extends BaseBackendImportService {
     public constructor(
         importServiceCollector: ImportServiceCollectorService,
         private repo: ParticipantControllerService,
-        private activeMeetingIdService: ActiveMeetingIdService,
+        private activeMeeting: ActiveMeetingService,
```

Comment:
I would prefer importing `ActiveMeetingService` directly in `ParticipantCsvExportService` and keeping this file as it is.